animations.js

```diff
@@ -30,4 +24,4 @@ let clickHandlerCircle = function () {
   }
 }
 
-circleIcon.addEventListener('click', clickHandlerCircle)
+circleIcon.addEventListener('click', clickHandlerCircle)
```

Comment:
Nice work doing the circle click event. A tip:

You can use `circleIcon.classList.toggle('active')` if not existing, it set active class, otherwise remove the it.

For now, it's totally okay to do 4 event listeners, one for each icon. Later we can DRY it too.